pages/plant/[id].js

```diff
@@ -0,0 +1,121 @@
+import Image from "next/image";
+import { useRouter } from "next/router";
+import styled from "styled-components";
+import BackLink from "@/components/BackLink";
+
+const StyledSeasonList = styled.ul`
+  list-style: none;
+  padding: 0;
+`;
+
+const IconsContainer = styled.article`
+  display: flex;
+  justify-content: space-around;
+`;
+
+const IconContainer = styled.div`
```

Comment:
We can relate with StephieTacks suggestion because the names of IconContainer (styled.div) and IconsContainer (styled.article) are very similar and could be a bit confusing.